packages/zoe/test/unitTests/contracts/vpool-xyk-amm/test-amm-governance.js

```diff
@@ -0,0 +1,351 @@
+    installBundle(zoe, autoswapBundle),
+    installBundle(zoe, contractGovernorBundle),
+    installBundle(zoe, committeeBundle),
+    installBundle(zoe, voteCounterBundle),
+  ]);
+  const installs = {
+    autoswap,
+    governor,
+    electorate,
+    counter,
+  };
+
+  const {
+    creatorFacet: committeeCreator,
+    instance: electorateInstance,
+  } = await E(zoe).startInstance(installs.electorate, {}, electorateTerms);
+
+  const governorTerms = {
+    timer,
+    electorateInstance,
+    governedContractInstallation: installs.autoswap,
+    governed: {
+      terms: ammTerms,
+      issuerKeywordRecord: { Central: centralR.issuer },
+      privateArgs: {},
+    },
+  };
+
+  const {
+    instance: governorInstance,
+    publicFacet: governorPublicFacet,
+    creatorFacet: governorCreatorFacet,
+  } = await E(zoe).startInstance(installs.governor, {}, governorTerms, {
+    electorateCreatorFacet: committeeCreator,
+  });
+
+  const ammCreatorFacetP = E(governorCreatorFacet).getInternalCreatorFacet();
+  const ammPublicP = E(governorCreatorFacet).getPublicFacet();
+
+  const [ammCreatorFacet, ammPublicFacet] = await Promise.all([
+    ammCreatorFacetP,
+    ammPublicP,
+  ]);
+
+  const g = { governorInstance, governorPublicFacet, governorCreatorFacet };
+  const governedInstance = E(governorPublicFacet).getGovernedContract();
+
+  const amm = { ammCreatorFacet, ammPublicFacet, instance: governedInstance };
+
+  return {
+    zoe,
+    installs,
+    governor: g,
+    amm,
+    committeeCreator,
+  };
+};
+
+const ammInitialValues = harden([
+  {
+    name: POOL_FEE_KEY,
+    value: 24n,
+    type: ParamType.NAT,
+  },
+  {
+    name: PROTOCOL_FEE_KEY,
+    value: 6n,
+    type: ParamType.NAT,
+  },
+]);
+
+test('amm change param via Governance', async t => {
+  const centralR = makeIssuerKit('central');
+  const electorateTerms = { committeeName: 'EnBancPanel', committeeSize: 3 };
+  const timer = buildManualTimer(console.log);
+
+  const ammTerms = {
+    timer,
+    poolFeeBP: POOL_FEE_BP,
+    protocolFeeBP: PROTOCOL_FEE_BP,
+    main: ammInitialValues,
+  };
+
+  const {
+    zoe,
+    amm,
+    committeeCreator,
+    governor,
+    installs,
+  } = await setupServices(electorateTerms, ammTerms, centralR, timer);
+
+  t.deepEqual(
+    await E(amm.ammPublicFacet).getGovernedParamsValues(),
+    {
+      main: {
+        PoolFee: {
+          name: POOL_FEE_KEY,
+          type: 'nat',
+          value: 24n,
+        },
+        ProtocolFee: {
+          name: PROTOCOL_FEE_KEY,
+          type: 'nat',
+          value: 6n,
+        },
+      },
+    },
+    'initial values',
+  );
+
+  const invitations = await E(committeeCreator).getVoterInvitations();
+  const { governorCreatorFacet } = governor;
+  const foo = await E(governorCreatorFacet).voteOnParamChange(
+    { parameterName: PROTOCOL_FEE_KEY },
+    20n,
+    installs.counter,
+    2n,
+  );
+  const { details } = foo;
```

Comment:
I suggest not bothering with `foo`:

```suggestion
  const { details } = await E(governorCreatorFacet).voteOnParamChange(
    { parameterName: PROTOCOL_FEE_KEY },
    20n,
    installs.counter,
    2n,
  );
```

Reply:
a vestige of debugging.   I should mark these temporary changes better so I find them more reliably.  I know: I'll add `foo` to my presubmit check.  (or if there are too many instance of `foo` in real code, I'll choose a temporary name I can check for.)